### PR TITLE
as.source helper

### PR DESCRIPTION
**Pull request by robertzk**
A little bit of cleanup highly recommended by [Hadley's style guide](http://adv-r.had.co.nz/Style.html). If you like it, I'll send more!

### CODE REVIEW

File: R/block.R [hunk 1]
```diff
@@ -112,6 +112,8 @@ block_exec = function(options) {
     }, add = TRUE)
   }
 
+  # This will be useful later on line 224 to extract what global variables
```
Comment (yihui) [inline R/block.R]: the line number can change, so please do not hard-code it

**Comment by yihui** [comment]
This one looks good (if you correct the minor issue I mentioned above).

It is a little "scary" to get ten pull requests all of a sudden! :) But I appreciate it.

File: R/block.R (hunk 1 above)
Comment (robertzk) [inline R/block.R]: Yeah I thought about this. I will remove it.

**Comment by robertzk** [comment]
How does it look now? 

I'm sorry about all the pull requests! Take your time, there is no hurry.

**Pull request merged**